envergo/evaluations/models.py

```diff
@@ -295,18 +295,16 @@ def create_version(self, author):
         version = EvaluationVersion(evaluation=self, created_by=author, content=content)
         return version
 
-    def render_content(self):
+    def render_content(self, context={}):
```

Comment:
I need to get the context from the view to display the sharing url on print. 
It is not very elegant and if this method is not called from a view, we won't have the information.
Any better idea ?